.gitattributes

```diff
@@ -0,0 +1,8 @@
+/tests export-ignore
```

Comment:
@stof, there's a similar issue with symfony/symfony itself by the way. I just noticed that there are a few deliberately broken xmls in */Tests/ (e.g. [invalid.xml in Config component](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Config/Tests/Fixtures/Util/invalid.xml)).

In that repository it might be more reasonable to also add .gitattributes, but remove slash before "tests". But you're right that there is a potential problem with namespaces.